index.js

```diff
@@ -131,10 +131,84 @@ newSightingForm.addEventListener('submit', function (event) {
     renderAnimalSightingPost(newSightingObject)
     newSightingForm.reset()
 
+    
 })
 
 
 /***** End of Starter Code *****/
 /************************** EVENTS PART 2 JS MINI CHALLENGE ******************************/
 
+/***** Deliverable 1 *****/
+
+// document.querySelector('ul#animals').addEventListener('click', function(event){
+//     if (event.target.matches)
+//     console.log('Like clicked!!!', event.target)
+
+// })
+
+/***** Deliverable 2 *****/
+
+// document.querySelector('button.delete-button').addEventListener('click', function(event){
+
+//     console.log('Delete clicked!!!', event.target)
+
+// })
+
+const ulAnimals = document.querySelector('ul#animals')
+
+    // console.log(ulAnimals)
+
+ulAnimals.addEventListener('click', function(event) {
+
+    // console.log(event.target)
+    if(event.target.matches('button.like-button')) {
+        // console.log('like clicked')
+        
+        const likesPtag = event.target.previousElementSibling
+        // console.log(likesPtag)
+        const likes = parseInt(likesPtag.textContent) + 1
+        likesPtag.textContent = `${likes} Likes`
+        // console.log(likesPtag)
+    }
```

Comment:
watch the syntax here. When you have an if/else statement it should look like this
```
if (condition) {
 // logic
} else if (condition) {
 //logic
}
```

`} else if (event.target.matches('button.delete-button')) {`